java/layer-wrapper/build.gradle.kts

```diff
@@ -10,7 +10,7 @@ dependencies {
     runtimeOnly(project(":awssdk-autoconfigure"))
 
     runtimeOnly("io.grpc:grpc-netty-shaded")
```

Comment:
Ah has been true of a couple of releases but now would be a good time to remove gRPC dependencies as they aren't needed anymore. Should improve cold start time